spec/index.bs

```diff
@@ -4600,21 +4610,34 @@ interface mixin GPUProgrammablePassEncoder {
             **Called on:** {{GPUProgrammablePassEncoder}} this.
 
             **Arguments:**
-            <!--The overload appears to be confusing bikeshed, and it ends up expecting this to
+            <pre class=argumentdef for="GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)">
+                |index|: The index to set the bind group at.
+                |bindGroup|: Bind group to use for subsequent render or compute commands.
+
+                <!--The overload appears to be confusing bikeshed, and it ends up expecting this to
                 define the arguments for the 5-arg variant of the method, despite the "for"
                 explicitly pointing at the 3-arg variant.-->
-            <!--pre class=argumentdef for="GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)">
-                index:
-                bindGroup:
-                dynamicOffsets:
-            </pre-->
+                <!--|dynamicOffsets|: Array containing buffer offsets in bytes for each entry in
+                    |bindGroup| with marked as {{GPUBindGroupLayoutEntry/hasDynamicOffset}}.-->
+            </pre>
 
             Issue: Resolve bikeshed conflict when using `argumentdef` with
                 {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)}}.
 
             **Returns:** void
 
-            Issue: Describe {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
+                    <div class=validusage>
+                        - |bindGroup| is [=valid=].
+                        - |bindGroup|.{{GPUObjectBase/[[device]]}} is |this|.{{GPUObjectBase/[[device]]}}.
+                        - |index| &lt; {{GPULimits/maxBindGroups|GPULimits.maxBindGroups}}.
+                        - dynamicOffsets.length is
```

Comment:
should be `|dynamicOffsets|.length`?

Reply:
This was because that arg wasn't documented in the argumentdef block due to a bikeshed bug, so making it a variable here would raise a "single use" warning. I updated the issue text to refer to the variable, though, which allows this to be marked properly now.